pkgs/applications/networking/cluster/pluto/default.nix

```diff
@@ -2,13 +2,13 @@
 
 buildGoModule rec {
   pname = "pluto";
-  version = "5.18.3";
+  version = "5.18.4";
 
   src = fetchFromGitHub {
     owner = "FairwindsOps";
     repo = "pluto";
     rev = "v${version}";
-    sha256 = "sha256-D85+cT4bRVQwyrXs+NZJetRIHP3I7nbJKqOTjatoxwc=";
+    sha256 = "sha256-/8ZJXy5FErLnnXpED0UL+xqOo4QZtmR1hpcSpVsE8mw=";
```

Comment:
```suggestion
    hash = "sha256-/8ZJXy5FErLnnXpED0UL+xqOo4QZtmR1hpcSpVsE8mw=";
```

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/254494/commits/7de036faedfb45e3220144111c0ed06325170a74